.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node_version: ['10', '12', '14', '15', '16']
+        node_version: ['12', '14', '15', '16', '17']
```

Comment:
maybe drop 15 too?

Reply:
I wish Node.js had usage statistics somewhere, but I think I'd like to keep v15 for now.